src/Validation/DoctrinePresenceVerifier.php

```diff
@@ -89,7 +89,11 @@ public function getMultiCount($collection, $column, array $values, array $extra
      */
     protected function select($collection)
     {
-        $em      = $this->getEntityManager($collection);
+        $em = $this->getEntityManager($collection);
+        if ($em === null) {
+            throw new \InvalidArgumentException("[$collection] is not a valid Doctrine type.");
```

Comment:
Can you move this if-statement to the `getEntityManager` method. It should only have to be implemented for `$this->registry->getManagerForClass($entity)` .

Change message to `No Entity Manager could be found for [%s]`.

Also import class + use sprintf.